### PR TITLE
add node.js sdk

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Relates to https://github.com/camunda/product-hub/issues/1045.

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by pepopowitz — APPROVED** [review]
This is great! 

If I get time today I'll also step through the quick-start on my own, to see if I run into any issues.

File: docs/apis-tools/node-js-sdk.md [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+Deployed process c8-sdk-demo
+```
+
+### Create a service worker
+
+Outside the main function, add the following code:
+
+```typescript
+console.log("Starting worker...");
+zbc.createWorker({
```
Comment (pepopowitz) [inline docs/apis-tools/node-js-sdk.md:214]:
```suggestion
zeebe.createWorker({
```

we declare this variable name as `zeebe`, not `zbc`.

File: docs/apis-tools/node-js-sdk.md [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+```typescript
+console.log(
+  `[Zeebe] Deployed process ${res.deployments[0].process.bpmnProcessId}`
+);
+```
+
+2. Inside the `main` function, add the following:
+
+```typescript
+const p = await zbc.createProcessInstanceWithResult({
```
Comment (pepopowitz) [inline docs/apis-tools/node-js-sdk.md:287]:
```suggestion
const p = await zeebe.createProcessInstanceWithResult({
```

we declare this variable name as `zeebe`, not `zbc`.

File: docs/apis-tools/node-js-sdk.md [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+## Connection configuration
+
+You have two choices:
+
+- Explicit configuration in code
+- Zero-configuration constructor with environment variables
+
+The best way to do the configuration is via the zero-configuration constructor in code, and all values for configuration supplied via environment variables. This allows you to test the same code against different environments with no code changes.
+
+The environment variables you need to set are as follows (replace with your secrets and URLs):
```
[the remark this answers is not in the document]
Comment (christinaausley) [reply docs/apis-tools/node-js-sdk.md]: Yes! Done 👍

**Review by pepopowitz — APPROVED** [review]
🙌

**Pull request merged**